### PR TITLE
backend scaffold

### PR DESCRIPTION
**Pull request by nunonamorado**
Just for showing a possible backend structure. I will start organizing stuff better, with the actual teampot schema.

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const { ApolloServer, gql } = require("apollo-server-express");
```
Comment (aye-dev) [inline backend/index.js]: I guess we need express for redirects, etc?

**Comment by aye-dev** [comment]
Looks good.  @manty let's focus on the frontend

**Comment by nunonamorado** [comment]
@aye-dev @Mvii7 are you going to develop in Typescript?

**Comment by Mvii7** [comment]
> @aye-dev @Mvii7 are you going to develop in Typescript?

Definetely

**Pull request closed without merging**